.github/workflows/integration_test.yaml

```diff
@@ -17,8 +37,3 @@ jobs:
       juju-channel: 3.4/stable
       channel: 1.28-strict/stable
       modules: '["test_charm", "test_nginx", "test_s3", "test_scaling", "test_matrix_auth"]'
-  allure-report:
```

Comment:
Why this removal? Is this already covered by the operator-workflows?

Reply:
This is only done in a scheduled event
https://github.com/canonical/synapse-operator/pull/642